discoverx/constants.py

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
Wouldn't it make sense to add this to the pydantic-based configuration instead of adding a new `Conf` class? We can add them as defaults. The `COLUMNS_TABLE_NAME` will become redundant as soon as we have save & load added.

Reply:
Refactored together into the DX class